src/css/components/_states.scss

```diff
@@ -0,0 +1,14 @@
+.is--centered {
```

Comment:
@felipprodrigues we can keep only one dash for states to avoid blending with modifiers